question/classes/external.php

```diff
@@ -314,4 +314,37 @@ public static function  get_random_question_summaries_returns() {
             )
         ]);
     }
+
+    /**
+     * Returns description of method parameters.
+     *
+     * @return external_function_parameters
+     */
+    public static function set_columnbank_order_parameters() {
+        return new external_function_parameters(
+                ['columns' => new external_value(PARAM_RAW, 'JSON String containing column order to set in config_plugins table')]
+        );
+    }
+
+    /**
+     * Returns description of method result value
+     * @return external_value
+     */
+    public static function set_columnbank_order_returns() {
+        return new external_single_structure(
+            ['status' => new external_value(PARAM_BOOL, 'status: true if success')]
+        );
+    }
+
+    /**
+     * Returns the columns plugin order
+     * @return string
```

Comment:
please add a new line `     *` before this line

Reply:
Solved